src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private StringBuilder stringBuilder = new StringBuilder();
```

Comment:
It's better to use StringBuilder stringBuilder as a local variables, because there is big potential of errors when you try to use your `getStatistic()` method of one object from different threads (multithreading).